packages/common/src/stores/ConfigStore.ts

```diff
@@ -10,11 +10,11 @@ type ConfigState = {
   config: Config;
   accessModel: AccessModel;
   settings: Settings;
-  integrationType: IntegrationType | null;
+  integrationType: string | null;
   supportedLanguages: LanguageDefinition[];
-  isSandbox: boolean;
-  clientId: string | null;
-  offers: string[];
+  // isSandbox: boolean;
+  // clientId: string | null;
+  // offers: string[];
```

Comment:
Just remove?

Reply:
Yes! I didn't cleanup yet because I wanted to get some reviews first :-)